src/js/disability-benefits/526EZ/tests/config/evidenceType.unit.spec.jsx

```diff
@@ -39,12 +39,32 @@ describe('Disability benefits 526EZ evidence type', () => {
         uiSchema={uiSchema}/>
     );
 
-    selectCheckbox(form, 'root_view:vaMedicalRecords');
-    selectCheckbox(form, 'root_view:privateMedicalRecords');
-    selectCheckbox(form, 'root_view:otherEvidence');
+    selectCheckbox(form, 'root_view:selectableEvidenceTypes_view:vaMedicalRecords', true);
+    selectCheckbox(form, 'root_view:selectableEvidenceTypes_view:privateMedicalRecords', true);
+    selectCheckbox(form, 'root_view:selectableEvidenceTypes_view:otherEvidence', true);
 
     form.find('form').simulate('submit');
-    expect(form.find('.usa-input-error').length).to.equal(0);
+    expect(form.find('.usa-input-error-message').length).to.equal(0);
```

Comment:
...and the class selector was partially missing. Confirmed on latest master that it's actually `.usa-input-error-message`

Reply:
:grimacing: Good catch!